### PR TITLE
Adding powering poe docs

### PR DESCRIPTION
**Pull request by Erol444**

### CODE REVIEW

**Comment by Luxonis-Brandon** [comment]
Thanks!

File: docs/source/pages/BW2098POE.rst [hunk 1]
```diff
@@ -34,6 +34,7 @@ Key features
 * On-board PCIe ref clk for BW2099 and PCIe/Ethernet bridge
 * User-selectable BW2099 boot configuration switches
 * Design files produced with Altium Designer 20
+* :ref:`PoE injector <Powerig PoE devices>` is required to power the device.
```
Comment (Luxonis-Brandon) [inline docs/source/pages/BW2098POE.rst:37]: Small type here on the section name.  Should be `Powering PoE devices`.  (The `n` is missing.)

File: docs/source/pages/powering_poe_devices.rst [hunk 2]
```diff
@@ -0,0 +1,37 @@
+Powerig PoE devices
```
Comment (Luxonis-Brandon) [inline docs/source/pages/powering_poe_devices.rst:1]: Missing `n` here.

**Review by cafemoloko — APPROVED** [review]
LGTM, thanks :)

**Pull request merged**